test/hub/profiles/profile-uri.spec.ts

```diff
@@ -79,7 +79,7 @@ makeSuiteCleanRoom('Profile URI Functionality', function () {
         ];
         expect(jsonMetadata.attributes).to.eql(expectedAttributes);
         expect(keccak256(toUtf8Bytes(tokenURI))).to.eq(
-          '0xe1731460db6ce5fa9f3a9f2bd778a8af49e623dceb531df6b1a5c162b7c2d79a'
+          '0xff9081b5ef994d2a060d3d34247641822fde6356f135f61a136298bb01a22958'
```

Comment:
I will try to explain why I did this and let me know what do you think.

In this case, `tokenURI` returns a base64-encoded metadata JSON. So I am decoding it, and asserting all the JSON fields except for the image one.

And this is just because the image is a base64-encoded SVG, it is difficult to check that the SVG is looking as expected. So my approach is to take the generated SVG, look that it renders fine in my browser, compute the expected hash and put it into the test assertion.

I see your point but I think that it's ok that the tests fails after the SVG gets altered, as it prevents not being noticed about a change that modifies the SVG image, and I expect that to force the dev to verify the new SVG is still looking good (to the human eyes).

However, if you think we have a better approach let me know.

Reply:
Oh yeah, I thought about having the SVG files and computing the hash on runtime. I can change it to follow that approach! I will update the PR later